### PR TITLE
Create Style Guide module

### PR DESCRIPTION
**Pull request by shuklina**
### How to review:
- [ ]  Make sure that Style Guide page is created.

### Steps for review:
- [ ]  Please enable `Open Y Style Guide` module;
- [ ]  Go to path `/styleguide`.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+/**
+ * Provides route responses for the Open Y Style Guide module.
+ */
+class OpenyStyleGuideController extends ControllerBase {
+
+  /**
+   * Renders content.
+   */
+  public function content() {
+    $query = \Drupal::service('entity.query')
```
Comment (miroslav-lee) [inline modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php:18]: Please, use DI
More see https://www.drupal.org/docs/8/api/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8

File: modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      ->get('menu_link_content')
+      ->condition('menu_name', 'style-guide');
+    $entity_ids = $query->execute();
+
+    /** @var \Drupal\Core\Entity\Sql\SqlContentEntityStorage $storageManager */
+    $storageManager = \Drupal::entityTypeManager()->getStorage('menu_link_content');
```
Comment (miroslav-lee) [inline modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php:24]: Please, use DI

File: modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    $activeTheme = \Drupal::service('theme.manager')->getActiveTheme();
```
Comment (miroslav-lee) [inline modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php:25]: Please, use DI

File: modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      $entity = $storageManager->load($id);
+      $value = $entity->get('link')->getValue();
+      $items[] = [
+        'title' => $entity->get('title')->value,
+        'link' => Url::fromUri($value[0]['uri']),
+      ];
+    }
+    if (empty($items)) {
+      return [
+        '#markup' => '<p>No Style Guide items found.</p>',
```
Comment (miroslav-lee) [inline modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php:38]: Please, use `$this->t()` for localization

File: modules/custom/openy_style_guide/templates/style-guide.html.twig [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <div class="row my-3">
+        <table class="table table-bordered">
+          <thead>
+          <tr>
+            <th scope="col">Component name</th>
```
Comment (miroslav-lee) [inline modules/custom/openy_style_guide/templates/style-guide.html.twig:8]: Use the construction `{{ 'Component name'|t }}` for simple text

**Comment by miroslav-lee** [comment]
@shuklina update branch and fix review comments, please

File: modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php (hunk 1 above)
Comment (shuklina) [reply modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php:18]: Thanks for the review comments, I will correct it.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
Way outdated and no demand
We switched to storybook

**Pull request closed without merging**